### ISSUE TITLE
a question about the parameter of equi model 

### ISSUE THREAD

**Issue by XiongLianga** [ethz-asl/kalibr#331]
In the support model's documents, equidistant (equi) (distortion_coeffs: [k1 k2 k3 k4]), there are four distortion parameters.  Does the k1 represent the coefficient of theta^3, and k2 represent the theta^5,etc？

**Comment by NikolausDemmel** [comment]
Yes, I think it should be like that. Same as opencv fisheye: https://docs.opencv.org/2.4/modules/calib3d/doc/camera_calibration_and_3d_reconstruction.html#fisheye

**Comment by kannidekan** [comment]
execuse me, do you know how could i get the IMU YAML file of the requirements in the camera-imu calibration? #341

Thanks very much!
